config/lib/jwtAuthentication.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
I think we can rename this file to `authorization`; makes more sense to me. If we wanted to support having multiple Authorization methods we could always create a folder named *authorization* & put individual auth strategies there; then naming this `jwt` would make sense.